### ISSUE TITLE
0.69: --no-backup is not working as expected anymore

### ISSUE THREAD

**Issue by migligli** [uncrustify/uncrustify#2340]
Version with the issue: _Uncrustify 0.69.0_
Version without the issue: _Uncrustify 0.68.1_

I ran uncrustify 0.69.0 in command line using the option _--no-backup_ on my C source files with the following command line:

`uncrustify.exe -c "C:\<pathToUsedConfig>\uncrustify.cfg" --no-backup "C:\<pathToFileToUncrustify>\file.c"`

If file.c needs to be uncrustifyed, the option _--no-backup_ will replace the file without a backup file, that is a copy of the original file before being modified by uncrustify. The issue is that 0.69.0 will produce a backup file suffixed with .uncrustify for each file (file.c.uncrustify). Even for files that are **NOT** modified by uncrustify. This issue is problematic since the source files are managed by a source control tool. And we don't want the noise produced by so many backup files. 

I tested the version 0.68.1 with the same command line as above. And there is no issue. No backup file is created. 

Does anyone know anything about this behavior with 0.69.0? Because of that, I cannot consider 0.69.0 for the moment.

**Comment by chris-mich** [comment]
I came into the same issue.  Adding the "--replace" option fixed my problem.

**Comment by migligli** [comment]
Yes indeed using "--replace" along with "--no-backup" fixes the problem. Of course, it is a temporary fix. I think the long term fix, that is to use "--no-backup" alone as before, has been merged in the branch _master_ but not yet released: #2326.
